README.md

```diff
@@ -58,6 +58,42 @@ Enjoy your Kubernetes cluster!
 [kvm]: https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine
 [ww-gitops]: https://www.weave.works/technologies/gitops/
 
+## Watch GipOps in action
+
+Now that we have a cluster installed, we can commit Kubernetes objects to the
+git repository and have them appear in the cluster. Let's add
+[podinfo][podinfo], an example Go microservice, to the cluster.
+
+[podinfo]: https://github.com/stefanprodan/podinfo
+
+```sh
+curl -fLo podinfo.yaml https://raw.githubusercontent.com/stefanprodan/podinfo/master/kustomize/deployment.yaml
```

Comment:
Note this only pulls the deployment.  If you want the service and HorizontalPodAutoscaler you can use:
```
kubectl apply --dry-run -k github.com/stefanprodan/podinfo//kustomize -o yaml > podinfo.yaml
```